docs/book/src/networking-introduction.md

```diff
@@ -0,0 +1,82 @@
+
+- **unreliable**: Transport packeten zijn er niet van verzekerd dat ze aan komen bij hun eindbestemming en op volgorde zijn. 
+- **reliable**: Transport packeten zijn er van verzekerd dat ze op volgorde op hun eindbestemming komen.
+
+Unreliability gives great uncertainty with a lot of freedom, while reliability gives great certainty with costs in speed and freedom.
+That is why some protocols such as QUIC, SCTP are built on UDP instead of TCP. 
+
+| Feature |  TCP  | UDP | QUIC
+| :-------------: | :-------------: | :-------------: | :-------------: |
+|  [Connection-Oriented][6]           |       Yes         | No                       | Yes
+|  Transport Guarantees               | Reliable          | Unreliable               | Reliable and/or unreliable with extension 
+|  Packet Transfer                    | [Stream-based][4] | Message based            | Message based and/or Stream based
+|  Header Size                        |  20 bytes         | 8 bytes                  |  ~16 bytes(depending on connection id)  
+|  [Control Flow, Congestion Avoidance/Control][5] | Yes  | No                       |  ** Yes, and possible controlled by userspace                                          
+|  Based On | [IP][3]                 | [IP][3]           |  UDP
+
+** QUIC control flow/congestion implementations will run in userspace wereas in TCP its running in kernelspace, 
+however there might be a kernel implementations for QUIC in the future.
+
+## 2. Issues with TCP 
+
+TCP has been around for a long time and was not designed with the modern internet in mind. 
+It has several difficulties that QUIC tries to resolve. 
+
+### Head-of-line Blocking
+
+One of the biggest issues with TCP is that of Head-of-line blocking. 
+It is a convenient feature because it ensures that all packages are sent and arrive in order. 
+However, in cases of high throughput (multiplayer game networking) and big load in a short time (web page load), this can severely impact latency.
+
+The issue is demonstrated in the following animation:
+
+![Head of line blocking][animation] 
+
+This animation shows that if a certain packet drops in transmission, all packets have to wait at the transport layer until it is resent by the other end.
+If the dropped packet is resent and arrived then all packets are freed from the transport layer. 
+
+Let's look at two areas where this head-of-line blocking issue creates issues. 
+
+**Web Networking**
+
+The last years websites have been growing in size which causes. 
+This increases the loading time and makes head-of-line blocking a more concerning topic. 
+To tackle this issue HTTP-2 has introduced a technique called multiplexing. 
+In short, this means that multiple TCP streams are initialized to communicate with the server. 
+It allows a server to transfer multiple sources in parallel over a single stream.
+
+**Multiplayer Game Networking**
+
+The web space is not the only area where this head-of-line blocking is a major concern.
```

Comment:
I still don't think it makes sense to talk about this here, since head-of-line blocking isn't really what causes the problems with datagrams these kinds of games need. Additionally "reliability" here seems to have an entirely different meaning than what has been discussed before.

Reply:
Hrm. What entirely different meaning do you see? Reliability and head of line blocking are two concepts interwoven concepts don't you think? Simply said, a drop of a single packet causes buffering which is not plausible in games.

Reply:
Reliability conceptually is about "am I sure the other side is going to see this data". Head-of-line blocking has more to do with ordering. In the world of TCP vs UDP, reliability and ordering are codependent, but conceptually I don't think that makes sense. If you needed reliability for a game transport, you could build a UDP-based transport protocol that does away with ordering, and it would have reliability without head-of-line blocking concerns.

Reply:
Okay, so we can remove it then?